scripts/e2e/init-template-e2e.js

```diff
@@ -117,9 +117,8 @@ async function initNewProjectFromSource(
 
     console.log('Running react-native init without install');
     execSync(
-      `node ./packages/react-native/cli.js init ${projectName} \
-        --directory ${directory} \
-        --template ${templatePath} \
+      `node @react-native-community/cli init ${projectName} \
+        --template @react-native-community/template \
```

Comment:
```suggestion
      --directory ${directory} \
      --template @react-native-community/template \
```

Reply:
You don't need `--template @react-native-community/template`.